update/update-version.md

```diff
@@ -1,139 +1,187 @@
 # Updating from v3.x.x to v4.x.x
 
-Starting from version v4.x.x, we have refactored the code for the VSI module, resulting in a significant change when transitioning from version 3.x.x to 4.x.x. This means that the Virtual Server Instances (VSIs) managed by this module will be deleted and recreated during the update process.
+Version v4.x.x changes the VSI module code in ways that result in significant changes when you update from version 3.x.x to 4.x.x. When you update, the Virtual Server Instances (VSIs) that are managed by this module are deleted and re-created.
 
-By employing the suggested method, you can avoid the need for VSI recreation.
+Follow these steps to avoid the need to re-create the VSIs when you update.
 
-- [Schematics](#schematics)
-- [Local Terraform State file](#local-terraform-state-file)
+## Before you begin
 
-## Schematics
+Make sure you have recent versions of these command-line prerequisites.
 
-If you have your cloud infrastructure deployed using Schematics, you can use the  `schematics_update_v3.x.x_to_v4.x.x.sh` script to avoid the recreation of Virtual Server Instances (VSIs).
+- [IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cli-getting-started)
+- [IBM Cloud CLI plug-ins](https://cloud.ibm.com/docs/cli?topic=cli-plug-ins):
+    - `is` plug-in (vpc-infrastructure)
+    - `sch` plug-in (schematics)
+- JSON processor `jq` (https://jqlang.github.io/jq/)
+- [Curl](). To test whether curl is installed on your system, run the following command:
 
-1. Make sure you have all the dependencies to run the script.
-    - IBM Cloud CLI
-    - IBM Cloud CLI 'is' plugin
-    - IBM Cloud CLI 'schematics' plugin
-    - jq
+    ```sh
+    curl -V
+    ```
+
+    If you need to install curl, see https://everything.curl.dev/install/index.html.
+
+
+## Select a procedure
+
+Select the procedure that matches where you deployed the code.
+
+- [Deployed with schematics](#deployed-with-schematics)
+- [Local Terraform](#local-terraform)
+
+## Deployed with schematics
+
+If you deployed your IBM Cloud infrastructure by using Schematics, you run the `schematics_update_v3.x.x_to_v4.x.x.sh` script as a Schematics job.
```

Comment:
its not run "as a Schematics job". The script runs and creates a Schematics job

Reply:
Also maybe it should hyperlink to the script here instead of using backticks?

Reply:
Going back and forth in my mind about whether we want to drive people to it before they look at the steps. Ended up adding,

> [View the script](https://file+.vscode-resource.vscode-cdn.net/Users/allen.dean/GH/TIM/terraform-ibm-landing-zone-vsi/update/schematics_update_v3.x.x_to_v4.x.x.sh).